client/src/components/Board/Task.tsx

```diff
@@ -0,0 +1,33 @@
+import { Typography, CardContent, Card, Box } from '@material-ui/core';
+import useStyles from './useStyles';
+import { Draggable } from 'react-beautiful-dnd';
+import { IPropTask } from '../../interface/Board';
+
+interface Props {
+  task: IPropTask;
+  index: number;
+}
+
+export default function Task({ task, index }: Props): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Draggable draggableId={task.id} index={index}>
+      {(provided, snapshot) => (
+        <Card
```

Comment:
WIth the dragging styles, it actually might be related to Card and not DND here as you are working with MUI Card. Perhaps we could just utilize a regular div to make the rotation work? 

Utilizing the library to the best of our ability is important but I think we can compromise to match our mock-up more. Give that a try and let me know how it works

Reply:
So switching the Card to a div didn't fix it, but I found that if I wrapped the Card in the div and moved the DND attributes to the div, then I could style the Card on drag and it works correctly.